packages/wrangler/src/index.tsx

```diff
@@ -864,8 +862,8 @@ export async function main(argv: string[]): Promise<void> {
         args.host ||
         config.dev.host ||
         (args.routes && args.routes[0]) ||
-        envRootObj.route ||
-        (envRootObj.routes && envRootObj.routes[0]);
+        config.route ||
+        (config.routes && config.routes[0]);
```

Comment:
extremely cool that we don't need to do the env dance anymore (and envRootObj was such a terribly named var 😅)

Reply:
🚀